source/Irrlicht/COpenGLDriver.cpp

```diff
@@ -3282,9 +3302,118 @@ void COpenGLDriver::enableClipPlane(uint32_t index, bool enable)
 		glDisable(GL_CLIP_DISTANCE0 + index);
 }
 
+void COpenGLDriver::initExtensions(bool stencilBuffer)
+{
+    core::stringc vendorString = (char*)glGetString(GL_VENDOR);
```

Comment:
you need to initExtensions after you init the function table for the main context

Reply:
because `glGetString` needs to be called from the pointer in the function table